### PR TITLE
Add Dropdown Menu to Slack Integration

### PR DESCRIPTION
**Pull request by benzuzu**
### Changes to Slack Integration

- Non-approve buttons have been moved to a dropdown menu for organization
- Clarification and Denial emails can be sent in this menu (ignore embargo for now)

Screen Shot 2021-07-14 at 11 41 37 AM
Screen Shot 2021-07-14 at 11 41 48 AM

this fixes https://github.com/oncokb/oncokb/issues/2655

### CODE REVIEW

File: src/main/java/org/mskcc/cbio/oncokb/service/MailService.java [hunk 1]
```diff
@@ -249,6 +249,11 @@ public void sendActiveTrialMail(UserDTO user, Boolean newAccountCreation) {
         sendEmailFromTemplate(user, MailType.ACTIVATE_FREE_TRIAL, context);
     }
 
+    @Async
+    public void sendAcademicClarificationEmail(UserDTO user) {
+        sendEmailWithLicenseContext(user, MailType.CLARIFY_ACADEMIC_NON_INSTITUTE_EMAIL, applicationProperties.getEmailAddresses().getLicenseAddress(), null, null);
```
Comment (zhx828) [inline src/main/java/org/mskcc/cbio/oncokb/service/MailService.java:254]: Here should be sent from `registrationAddress`

File: src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java [hunk 2]
```diff
@@ -120,6 +128,44 @@ public void sendConfirmationOnUserAcceptsTrialAgreement(UserDTO userDTO, Instant
...
+                        }
+                    }
+                }
+            }
+        } catch (IOException | SlackApiException e) {
+            log.error("error: {}", e.getMessage(), e);
+        }
+        return userList;
+    }
+
```
Comment (zhx828) [inline src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java:168]: This is just rearrangement, nothing changed?

**Review by zhx828 — CHANGES_REQUESTED** [review]
Some minor comments

File: src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java (hunk 2 above)
Comment (benzuzu) [reply src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java:168]: Yes, double checked just now.

**Review by zhx828 — APPROVED** [review]
👍🏼

**Pull request merged**